.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release maven package
+on:
+  workflow_dispatch:
+
+env:
+    SIGN_KEY: ${{ secrets.SIGN_KEY }}
+    SIGN_KEY_ID: ${{ secrets.SIGN_KEY_ID }}
+    SIGN_KEY_PASS: ${{ secrets.SIGN_KEY_PASS }}
+
+jobs:
+  release_maven_package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate branch
+        run: |
+          echo "Release can only be triggered from master branch"
+          exit 1
+          if: ${{ github.ref != 'refs/heads/master' }}
+
+      - uses: actions/checkout@v3
+
+      - name: Set up jdk
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '8'
```

Comment:
We need to setup the Sonatype server here to ensure we can deploy. See: https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven#publishing-packages-to-the-maven-central-repository